sql/src/main/java/io/crate/operation/scalar/ExtractFunctions.java

```diff
@@ -106,12 +112,61 @@ public static FunctionInfo functionInfo(Extract.Field field) {
 
     private static FunctionInfo createFunctionInfo(Extract.Field field) {
         return new FunctionInfo(
-            new FunctionIdent(String.format(Locale.ENGLISH, NAME_TMPL, field.toString()), ARGUMENT_TYPES),
+            new FunctionIdent(NAME_PREFIX + field.toString(), ARGUMENT_TYPES),
             DataTypes.INTEGER,
             FunctionInfo.Type.SCALAR
         );
     }
 
+    private static class ExtractFunction extends Scalar<Object, Object> implements FunctionFormatSpec {
```

Comment:
Please add some javadoc here or on ExtractFunctions to explain the new delegating functionality.